configs/config.hs

```diff
@@ -52,6 +52,12 @@ writeMailtoFile m = do
   B.writeFile fname m
   pure ""
 
+fromMail :: [Mailbox]
+fromMail =
+    [ Mailbox
+          (Just "Joe Bloggs")
+          (AddrSpec "joe" (DomainDotAtom $ pure "foo.test"))
```

Comment:
I'd prefer you created the list with multiple domain parts, i.e.
```haskell
"foo" :| ["test"]
```

If you don't want to import the `NonEmpty((:|))` constructor, you can use `pure` multiple times:
```haskell
pure "foo" <> pure "test"  -- I'm cool with this.
```

Reply:
`NonEmpty` is in *base*.  But as I said I don't mind either way.